keylime/cmd/create_policy.py

```diff
@@ -531,6 +543,8 @@ def main() -> None:
             ignored_keyrings = base_policy.get("ima", {}).get("ignored_keyrings", [])
             policy["ima"]["ignored_keyrings"] = ignored_keyrings
             policy["verification-keys"] = base_policy.get("verification-keys", "")
+            if base_policy.get("rejects", None) is not None:
+                policy["rejects"] = base_policy.get("rejects", {})
```

Comment:
Why not to create v2 policies here? Seems that `policy["rejects"] = base_policy.get("rejects", {})` will do the right thing is "rejects" is missing from the base_policy

Reply:
Ok, will fix this.